toolchain/check/testdata/builtins/int/make_type_signed.carbon

```diff
@@ -78,7 +78,7 @@ library "[[@TEST_NAME]]";
 
 import library "types";
 
-fn Negate(n: i32) -> i32 = "int.snegate";
+fn Negate(a: IntLiteral()) -> IntLiteral() = "int.snegate";
```

Comment:
I thought changes to tests were being split off into a separate PR?

Reply:
They are in https://github.com/carbon-language/carbon-lang/pull/5683